src/services/LoadService.js

```diff
@@ -1,11 +1,51 @@
-import { networks } from "../store/store";
-import { nodes } from "../store/store";
-import { main_config } from "../store/store";
+import { nodes, networks, max_at } from "../store/scenario";
 // import { max_at } from "../store/store";
 
-export default function loadConfig(conf) {
+export function clearAll() {
+  networks.update(_ => {});
+  nodes.update(_ => {});
+  max_at.update(_ => 0);
+}
+
+
+
+export function initNetworks() {
+  networks.update(val => {
+    let tmp = val;
+    tmp[-1] = {
+      id: -1,
+      color: "#000000",
+      ssid: "default",
+      type: null,
+      addr: null
```

Comment:
prečo tam je táto sieť vlastne?

Reply:
lebo farba nody sa zoberie cez `$networks[node.l2id]`. v pripade, ze zmazem network s id `l2id`, sa nastavi color na `undefined` a bude to hadzat errory. takze dame dakuto "hidden siet" s id -1 aby to nepadalo a teda ked zmazeme siet v ktorej su nody, tak sa nastavi l2id na -1 a nepadne to

Reply:
🤔